security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,24 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-6"]
+== OpenShift File Integrity Operator 1.3.6
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.6:
+
+* link:https://access.redhat.com/errata/RH??-2025:????[RH??-2025:???? OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-1-3-6-bug-fixes"]
+=== Bug fixes
+
+* Previously, running `oc annotate fileintegrities/ file-integrity.openshift.io/re-init-on-failed=` would trigger a reinitialization on all nodes. Now, it only reinitializes the nodes where failures occurred.
```

Comment:
Sorry, the command should be 
`oc annotate fileintegrities/<fileintegrity-name> file-integrity.openshift.io/re-init-on-failed=`
the <fileintegrity-name> is the name of the fileintegrity.